tests/integration/experiments/test_tablets.py

```diff
@@ -29,7 +38,7 @@ def verify_same_host_in_tracing(self, results):
             LOGGER.info("TRACE EVENT: %s %s %s", event.source, event.thread_name, event.description)
             host_set.add(event.source)
 
-        assert len(host_set) == 1
+        assert len(host_set) in [1, 2]
         assert 'locally' in "\n".join([event.description for event in events])
 
```

Comment:
❓ Why did you edit this method (`verify_same_host_in_tracing`)? As far as I can tell it is not used in the new tests that you wrote.

Reply:
I have changed replication factor, now each tablet gets two replicas.
And now writes gets two hosts in the trace, while reads - one.

Reply:
Ok, makes sense